package.json

```diff
@@ -5,23 +5,16 @@
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
```

Comment:
I noticed that in js-textile, this was "dist/index". any reason for the difference?

Reply:
yeh we should leave off the suffix

Reply:
But they aren't used here anyway, so will just remove them